x-pack/plugins/security_solution/common/endpoint/data_loaders/index_endpoint_hearbeats.ts

```diff
@@ -79,7 +79,8 @@ export const indexEndpointHeartbeats = async (
   const operations = docs.concat(unbilledDocs).flatMap((doc) => [
     {
       index: {
-        _index: ENDPOINT_HEARTBEAT_INDEX,
+        // simulating different namespaces
+        _index: `.logs-endpoint.heartbeat-${doc.agent.id}`,
```

Comment:
Just in case things are failing - the namespace is normally restricted as to what characters it can have (ex. no `-` dash).